rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -310,6 +302,23 @@ async fn smoke_test(ctlr_tx: CtlrTx) -> Result<()> {
         .await
         .context("Failed to send ClearLogs request")?;
 
+    // Tray icon stress test
```

Comment:
what's the purpose of this stress test?

Reply:
There is a strange part deep inside Tauri that exports the icon as a PNG to `/run` _every time_ we call `set_icon`: https://github.com/tauri-apps/tao/blob/tao-v0.16.7/src/platform_impl/linux/system_tray.rs#L119

I wanted to make sure this doesn't take too long or use too much disk. It looks fine, and `/run` is supposed to be a RAM disk, so it won't use real disk space. But in case there's a performance regression or something, it could show up here.